core/templates/dev/head/pages/exploration_editor/editor_tab/TrainingDataServiceSpec.js

```diff
@@ -234,68 +187,36 @@ describe('TrainingDataService', function() {
 
     // Verify initial state.
     var state = ess.getState('State');
-    expect(state.interaction.answerGroups[0].rules[1]).toEqual(
-      rof.createNew(RULE_TYPE_CLASSIFIER, {
-        training_data: ['text answer']
-      })
-    );
+    expect(state.interaction.answerGroups[0].training_data).toEqual([
+      'text answer'
+    ]);
     expect(state.interaction.confirmedUnclassifiedAnswers).toEqual([
       'second answer'
     ]);
 
     // Ensure emptying the default unclassified answers is handled properly.
     tds.trainAnswerGroup(0, 'second answer');
     state = ess.getState('State');
-    expect(state.interaction.answerGroups[0].rules[1]).toEqual(
-      rof.createNew(RULE_TYPE_CLASSIFIER, {
-        training_data: ['text answer', 'second answer']
-      })
-    );
+    expect(state.interaction.answerGroups[0].training_data).toEqual([
+      'text answer', 'second answer'
+    ]);
     expect(state.interaction.confirmedUnclassifiedAnswers).toEqual([]);
 
     // Ensure emptying the answer group's classifier properly deletes the rule
```

Comment:
Are these comments still correct? Should classifier be "training data"? (I'm not sure, just wanted to check that you've looked at them.)

Reply:
Nope, the test case itself isn't needed anymore as we don't have a rule.

Reply:
OK, let's make sure to update the comments / delete stuff that's not needed then.

Reply:
Done